site/content/en/docs/Guides/Backfill/_index.md

```diff
@@ -19,34 +19,32 @@ Some scenarios that could happen when playing multiplayer games include:
+5. Director starts allocating a Game Server with Backfill `B1` information.
+6. Player 2 creates a Ticket (`T2`).
 7. Director calls `Backend.FetchMatches`.
 8. MMF runs `QueryBackfills` and `QueryTickets` using the MatchProfile. They return `B1` and `T2` accordingly.
-9. MMF determines `B1` could be used because it has open slots available. `T2` and `B1` forms a new Match.
-10. When GameServer allocating ends, it starts polling Open Match to acknowledge the backfill with `Frontend.AcknowledgeBackfill` by ID received in the Match (`B1.ID`) and in order to assign the address of the GameServer for `T1` and `T2`.
-
-*: The Backfill should be created with field `AllocateGameServer` set to `true`, so GameServer orchestrator (e.g.: [Agones](https://agones.dev/site/)) knows it has to create a new GameServer.
+9. MMF determines `B1` could be used based on data set on `B1` by the last run of the MMF (e.g. a number of open slots). `T2` and `B1` form a new Match.
+10. When the Game Server has started, it begins polling Open Match to acknowledge the backfill with `Frontend.AcknowledgeBackfill` using the backfill's ID (`B1.ID`) and supplies connection information for the server, to be returned as the `Assignment` data of the tickets `T1` and `T2`. The Game Server also receives the ticket data of the tickets that were assigned so that any data supplied on the ticket (e.g. player IDs) may be used by the game server.
 
 ## Considerations
 
-In order to use Backfill you need to customize your MMF and your Game Servers. BackfillId should be propagated to each Game Server during Game Server allocation. Which would be used later in `AcknowledgeBackfill` requests to define Backfill which is tied to this Game Server.
+In order to use Backfill you need to customize your MMF and your Game Servers. Backfill IDs should be propagated to each Game Server during Game Server allocation to be used in `AcknowledgeBackfill` requests to define Backfill which is tied to the Game Server.
+
 If Backfill is not being used, no changes are required since the Backfill API doesn't change 
 the behavior of the current Open Match API state.
 
 ### Acknowledging Backfills
 
-AcknowledgeBackfill is a request, that acts as a periodic ping, made to Open Match Frontend to set the assignment on the tickets associated with this backfill.
-GameServer receives current Backfill status as a response.
+`AcknowledgeBackfill` is a request that acts as a periodic heartbeat and is made to Open Match Frontend to set the assignment on the tickets associated with this backfill. The Game Server receives the current Backfill status as well as a list of new tickets that were just acknowledged by the request as a response. The list is not idempotent, but can be appended to prior lists received from `AcknowledgeBackfill`.
```

Comment:
Can we have a different word for heartbeat for this context?

Reply:
Sure, but I'd be interested in the reason!

Reply:
Wording changed, though.

Reply:
I've followed up offline in Slack